src/cli/update-strict-comments/getFilePaths.ts

```diff
@@ -0,0 +1,23 @@
+import { isFileStrictByPath } from '../../common/isFileStrictByPath';
+import { getAbsolutePath } from '../../common/getAbsolutePath';
+import { findStrictErrors } from '../findStrictErrors';
+
+export const getFilePathsWithErrors = async (allFilePaths: string[]) => {
+  const errors = await findStrictErrors(allFilePaths);
+
+  const getFilePathFromErrorMessage = (error: string) =>
+    getAbsolutePath(process.cwd(), error.split('(')[0]);
+
+  return [...new Set(errors.map(getFilePathFromErrorMessage))];
+};
+
+// Returns an array of file paths that are on config path and do not contain strict errors
+export const getFilePathsWithoutErrors = (
+  allFilePaths: string[],
+  filePathsWithErrors: string[],
+  configPaths?: string[],
+) =>
+  allFilePaths.filter(
+    (filePath) =>
+      !isFileStrictByPath(filePath, configPaths) && !filePathsWithErrors.includes(filePath),
```

Comment:
i think that if we would convert `filePathsWithErrors` to set before filtering it could be a littlebit faster, but i dont know if its worth optimizing

Reply:
Set doesn't have a `filter` method unfortunately

Reply:
I was talking about something like this
```ts
export const getFilePathsWithoutErrors = (
  allFilePaths: string[],
  filePathsWithErrors: string[],
  configPaths?: string[],
) =>
  const filePathsWithErrorsSet = new Set(filePathsWithErrors);
  allFilePaths.filter(
    (filePath) =>
      !isFileStrictByPath(filePath, configPaths) && !filePathsWithErrorsSet.has(filePath),
  );
```